七牛上传时候增加参数的支持

tammybi opened the pull request.

tammybi committed:
七牛上传时候增加参数的支持

yinwm commented on src/main/java/com/yotouch/base/util/QiniuUtil.java:
```diff
@@ -64,9 +64,9 @@ private String getOverwriteToken(String bucket, String key) {
     }
 
-    public String upload(String name, byte[] content) throws IOException {
+    public String upload(String name, byte[] content, StringMap params) throws IOException {
```
`upload(String name, byte[] content)` 这个签名的方法是 `public` 的， 这个 Class 本身是 `@Component`。说明这个方法很有可能已经被其他不知道什么地方的代码使用了。那么直接修改方法签名这事儿是绝对不允许的，这样就意味着很多未知的代码在升级 pylon 之后就歇逼了。

tammybi committed:
public 的方法，很可能在其他地方也被用到了，所以不能随便的修改方法本身，而是需要考虑到兼容的问题。
public

yinwm commented on src/main/java/com/yotouch/base/util/QiniuUtil.java:
```diff
@@ -85,8 +85,12 @@ public String upload(String name, byte[] content) throws IOException {
         }
     }
 
+    public String upload(Entity att, StringMap params) throws IOException {
```
StringMap 这部分到底是什么意思？我指的是从业务逻辑上。这东西是一个 qiniu 的概念吧，那么是否意味着，我想使用一些功能的时候，比如压缩啊、转码啊，我必须读取七牛的文档才行。能否直接给出来一些方便的方法，让用户无需知道很多细节。比如 `uploadImage(Entity att)` 是上传图片，那么 `uploadImageWithCompress(Entity att, int maxSize)` 这样就是压缩图片。其他的类似 `uploadAudio(Entity att)` `uploadAudioToMp3`

tammybi committed:
这个掉了。还是因为public的方法，有可能已经被用过了。所以不能直接改现有的方法。

tammybi committed:
音频转换成mp3格式的，并且接收七牛的推送结果

tammybi committed:
七牛的转码参数添加，以及七牛通知的url设置

tammybi committed:
修改文件名的命名方式。之前用的错了。  java.sql 这个包，是根据数据库相关的东西的。
https://worktile.com/project/ba7c12e4074c4734b721c229769a1292/task/489e772b4dae4c40bada16d8d5b5f258
https://worktile.com/project/ba7c12e4074c4734b721c229769a1292/task/94cf392697ff4175830daa16d5b6f060

yinwm commented:
先合了，其实 convertToMp3 这个方法用起来还是别扭
